src/api/app/views/webui2/webui/monitor/_workers_table.html.haml

```diff
@@ -23,12 +23,12 @@
 - else
   .d-flex.flex-wrap
     - workers_sorted.each do |name, hash|
-      %ul.monitorboxrow.list-unstyled.overflow-hidden
-        %li.builderbox.float-left
+      %ul.monitorboxrow.list-unstyled.overflow-hidden.m-3
+        %li.builderbox.float-left.mt-3
```

Comment:
The `mt-3` class is not applied because in [monitor.css](https://github.com/openSUSE/open-build-service/pull/7784/files#diff-1f4af4f45f82c9a3610f8c9459c85564R16) it's defined a `margin-top: 3px !important`.

This [border](https://github.com/openSUSE/open-build-service/pull/7784/files#diff-1f4af4f45f82c9a3610f8c9459c85564R12) could also be removed because it as none effects.